pkg/security/apis/security/v1/types.go

```diff
@@ -81,6 +81,9 @@ type SecurityContextConstraints struct {
 	// used to generate a value for a pod the first non-wildcard profile will be used as
 	// the default.
 	SeccompProfiles []string `json:"seccompProfiles,omitempty" protobuf:"bytes,20,opt,name=seccompProfiles"`
+	// AllowedFlexDrivers is a whitelist of allowed Flexvolume drivers.
+	// Empty or nil indicates that all drivers may be used.
+	AllowedFlexDrivers []string `json:"allowedFlexDrivers" protobuf:"bytes,21,opt,name=allowedFlexDrivers"`
```

Comment:
So this field is used when the user already has flexVolume power right?  If they don't, then this is never used?

If that's the case, I'd expect a validation rule and some more detail in the doc.  Also, why not move up underneath `Volumes`?  The proto number can still be 21 up there.